code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -176,7 +176,17 @@
 	var/datum/language/species_language = GLOB.all_languages[language]
 	return species_language.get_random_name(gender)
 
-/datum/species/proc/create_organs(mob/living/carbon/human/H) //Handles creation of mob organs.
+/datum/species/proc/create_organs(mob/living/carbon/human/H, includes_special_organs = FALSE) //Handles creation of mob organs.
+	var/list/special_organs = list()
+	if(includes_special_organs)
+		for(var/obj/item/organ/internal/I in H.internal_organs)
+			if(I.special)
+				if((I.status & ORGAN_DEAD) && (I.status & ORGAN_ROBOT)) // cybernetic + broken
```

Comment:
But dead organic organs stay?...

Reply:
Dead organic organs sting, as the changeling can heal all organic organs.

Changelings can not heal dead robotic organs, so they are discarded